### PR TITLE
Example to demonstrate BertTokenizer as a customop

### PR DESCRIPTION
**Pull request by shaahji**
A simple example to demonstrate loading of BertTokenizer as a customop
in a onnx model

### CODE REVIEW

File: tutorials/bert_customop.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# Load the onnx model as encoded string
+content = open(str(model_path), "rb").read()
+onnx_model_b64 = base64.b64encode(base64.decodebytes(content))
+
+# Load the corresponding vocabulary
+vocab_filepath = "data/bert-base-cased.txt"
+vocabulary = open(vocab_filepath, 'r', encoding='utf-8').read()
+
+# Create a new node for the custom op
+customop_node = onnx.helper.make_node(
```
Comment (wenbingl) [inline tutorials/bert_customop.py:34]: Why do we use ONNX API, instead of pnp.export here?

**Comment by wenbingl** [comment]
What's the function difference from bert_e2e.py in the same folder?

**Comment by shaahji** [comment]
> What's the function difference from bert_e2e.py in the same folder?

bert_e2e is an example demonstrating use of bert with pre & post processing steps.
This example is demonstrating use of bert tokenizer as a custom op.

This example was requested by the AutoML team.

File: tutorials/bert_customop.py (hunk 1 above)
Comment (shaahji) [reply tutorials/bert_customop.py:34]: We are loading the model only as a base64 encoded data. We aren't really exporting the model here to disk. We are merely building it in memory.

**Comment by shaahji** [comment]
Discard!

**Pull request closed without merging**